common/JailUtil.cpp

```diff
@@ -168,12 +171,17 @@ void cleanupJails(const std::string& root)
         for (const auto& jail : jails)
         {
             const Poco::Path path(root, jail);
+            // Delete "tmp" directory after cleaning all the jails
+            if (jail == "tmp")
+                continue;
             // Delete tmp and link cache with prejudice.
```

Comment:
Please amend the comment.

Reply:
"Amend" in a sense "explain why it is needed"?

Reply:
Is it better now?